app/Http/Controllers/JobController.php

```diff
@@ -168,6 +166,121 @@ function ($value, $key) {
 
         $jobLang = Lang::get('applicant/job_post');
 
+        $gocLang = Lang::get('common/goc');
+
+        $skillsLang = Lang::get('common/skills');
+
+        $skillsArray = [];
+        foreach ($essential as $criterion) {
+            $skillsArray[] = [
+                'skill' => $criterion['skill']['name'],
+                'level' => $skillsLang['skill_levels'][$criterion
```

Comment:
This is an older way of getting the skill level name. You can use the more readable `$criterion->level_name` which makes use of the `getLevelNameAttribute()` [accessor](https://laravel.com/docs/8.x/eloquent-mutators#introduction) on the Criteria.php model.

Reply:
Thanks @tristan-orourke. I have updated this to use the accessor with [this commit](https://github.com/GCTC-NTGC/TalentCloud/pull/4537/commits/7c2f0bed953dbd1d2b8c04d659c47f8f517f69df).